tests/components/romy/test_config_flow.py

```diff
@@ -0,0 +1,195 @@
+"""Test the ROMY config flow."""
```

Comment:
Please assert also the unique id in the tests

Reply:
assert added